vector/src/main/AndroidManifest.xml

```diff
@@ -338,7 +412,8 @@
 
         <service
             android:name=".features.call.telecom.VectorConnectionService"
-            android:permission="android.permission.BIND_TELECOM_CONNECTION_SERVICE">
+            android:permission="android.permission.BIND_TELECOM_CONNECTION_SERVICE"
+            android:exported="true">
```

Comment:
Maybe this will also work with exported="false" ?

Reply:
Can you test it please?

Reply:
I think **VectorConnectionService** is not even working so there will be no problem to set `exported=false`.  I also test incoming and outgoing calls and they work properly.

Reply:
ok, thanks!